File: exs/mundo_1/python/016.py
"""
Desafio 016

Problema: Crie um programa que leia um número Real qualquer
          pelo teclado e mostre na tela a sua porção Inteira.

Resolução do problema:
"""
import math

num = float(input('Informe um número flutuante: '))
print(math.trunc(num))  # Metodo floor() tem o mesmo resultado

File: exs/mundo_1/javascript/016.js
/*
Desafio 016

Problema: Crie um programa que leia um número real qualquer 
          pelo teclado e mostre na tela a sua porção inteira

Resolução do problema:
*/

const input = require('readline-sync').question

let num = Number(input('Informe um número flutuante: '))

console.log(Math.trunc(num));